src/command_modules/azure-cli-security/azure/cli/command_modules/security/commands.py

```diff
@@ -0,0 +1,176 @@
+        client_factory=cf_security_auto_provisioning_settings,
+        operation_group='security_auto_provisioning_settings'
+    )
+
+    security_discovered_security_solutions_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.discovered_security_solutions_operations#DiscoveredSecuritySolutionsOperations.{}',
+        client_factory=cf_security_discovered_security_solutions,
+        operation_group='security_discovered_security_solutions'
+    )
+
+    security_external_security_solutions_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.external_security_solutions_operations#ExternalSecuritySolutionsOperations.{}',
+        client_factory=cf_security_external_security_solutions,
+        operation_group='security_external_security_solutions'
+    )
+
+    security_jit_network_access_policies_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.jit_network_access_policies_operations#JitNetworkAccessPoliciesOperations.{}',
+        client_factory=cf_security_jit_network_access_policies,
+        operation_group='security_jit_network_access_policies'
+    )
+
+    security_locations_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.locations_operations#LocationsOperations.{}',
+        client_factory=cf_security_locations,
+        operation_group='security_locations'
+    )
+
+    security_pricings_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.pricings_operations#PricingsOperations.{}',
+        client_factory=cf_security_pricings,
+        operation_group='security_pricings'
+    )
+
+    security_topology_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.topology_operations#TopologyOperations.{}',
+        client_factory=cf_security_topology,
+        operation_group='security_topology'
+    )
+
+    security_workspace_settings_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.workspace_settings_operations#WorkspaceSettingsOperations.{}',
+        client_factory=cf_security_workspace_settings,
+        operation_group='security_workspace_settings'
+    )
+
+    with self.command_group('security task',
+                            security_tasks_sdk,
+                            client_factory=cf_security_tasks) as g:
+        g.custom_command('list', 'list_security_tasks')
+        g.custom_command('show', 'get_security_task')
+
+    with self.command_group('security alert',
+                            security_alerts_sdk,
+                            client_factory=cf_security_alerts) as g:
+        g.custom_command('list', 'list_security_alerts')
+        g.custom_command('show', 'get_security_alert')
+        g.custom_command('dismiss', 'dismiss_security_alert')
+        g.custom_command('activate', 'activate_security_alert')
```

Comment:
Can these be incorporated into an `update` command? This will be more consistent with the rest of the CLI commands.

Reply:
updated